test/lib/builder/builder.js

```diff
@@ -572,6 +594,41 @@ const applicationGTree = {
 	}
 };
 
+const applicationGTreeWithExcludes = {
+	"id": "application.g",
+	"version": "1.0.0",
+	"path": applicationGPath,
+	"_level": 0,
+	"specVersion": "0.1",
+	"type": "application",
+	"metadata": {
+		"name": "application.g",
+		"namespace": "application/g",
+		"copyright": "Some fancy copyright"
+	},
+	"dependencies": [],
+	"resources": {
+		"configuration": {
+			"paths": {
+				"webapp": "webapp"
+			}
+		},
+		"pathMappings": {
+			"/": "webapp"
+		}
+	},
+	"builder": {
+		"resources": {
+			"excludes": [
+				"/subcomponentA/**",
+				"!**/manifest.json",
```

Comment:
So this will re-include all `manifest.json` files, right?

Reply:
Yes